ontology/yaml/resources/METERS/entity_types/GM.yaml

```diff
@@ -32,3 +32,10 @@ GM_PULSE:
   uses:
   - gas_flowrate_sensor
   - gas_volume_accumulator
+
+GM_GV:
```

Comment:
Use `GM_STANDARD` or `GM_PULSE` and flag any missing points rather than creating new abstract and entity types.